### PR TITLE
update data.js to include Yurui (me)

### PR DESCRIPTION
**Pull request by pallymore**

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -2064,6 +2064,36 @@ const pages = [
...
     ],
   },
+    {
+    // Your name
+    name: 'Yurui Zhang',
+    // Short description
+    description:
+      'Full-stack developer. Dark mode enthusiast. Quality software devotee.',
+    // URL to your /uses page
+    url: 'https://www.linkedin.com/in/yuruiology/',
```
Comment (HugoDF) [inline src/data.js]: url needs to be a "uses" page

File: src/data.js [hunk 2]
```diff
@@ -2064,6 +2064,36 @@ const pages = [
       'YouTuber',
     ],
   },
+    {
+    // Your name
```
Comment (HugoDF) [inline src/data.js]:
```suggestion
```

File: src/data.js [hunk 3]
```diff
@@ -2064,6 +2064,36 @@ const pages = [
...
+    name: 'Yurui Zhang',
+    // Short description
```
Comment (HugoDF) [inline src/data.js]:
```suggestion
```

File: src/data.js [hunk 4]
```diff
@@ -2064,6 +2064,36 @@ const pages = [
...
+    description:
+      'Full-stack developer. Dark mode enthusiast. Quality software devotee.',
+    // URL to your /uses page
```
Comment (HugoDF) [inline src/data.js]:
```suggestion
```

File: src/data.js [hunk 5]
```diff
@@ -2064,6 +2064,36 @@ const pages = [
...
+    url: 'https://www.linkedin.com/in/yuruiology/',
+    twitter: '@yuruiology',
+    // An emoji that describes you
+    emoji: '🐙',
+    // emoji of your country's flag
```
Comment (HugoDF) [inline src/data.js]:
```suggestion
```

File: src/data.js [hunk 6]
```diff
@@ -2064,6 +2064,36 @@ const pages = [
...
+    {
+    // Your name
+    name: 'Yurui Zhang',
+    // Short description
+    description:
+      'Full-stack developer. Dark mode enthusiast. Quality software devotee.',
+    // URL to your /uses page
+    url: 'https://www.linkedin.com/in/yuruiology/',
+    twitter: '@yuruiology',
+    // An emoji that describes you
```
Comment (HugoDF) [inline src/data.js]:
```suggestion
```

File: src/data.js [hunk 7]
```diff
@@ -2064,6 +2064,36 @@ const pages = [
...
+    emoji: '🐙',
+    // emoji of your country's flag
+    country: '🇨🇳',
+    // ONE of: apple, windows or linux
+    computer: 'apple',
+    // iphone or android
+    phone: 'iphone',
+    // Tags - You can add your own, but please keep it to one word. "Social vape entrepreneur influencer denver" isn't a tag.
```
Comment (HugoDF) [inline src/data.js]:
```suggestion
```

File: src/data.js [hunk 8]
```diff
@@ -2064,6 +2064,36 @@ const pages = [
...
+    // URL to your /uses page
+    url: 'https://www.linkedin.com/in/yuruiology/',
+    twitter: '@yuruiology',
+    // An emoji that describes you
+    emoji: '🐙',
+    // emoji of your country's flag
+    country: '🇨🇳',
+    // ONE of: apple, windows or linux
+    computer: 'apple',
+    // iphone or android
```
Comment (HugoDF) [inline src/data.js]:
```suggestion
```

File: src/data.js [hunk 9]
```diff
@@ -2064,6 +2064,36 @@ const pages = [
...
+    description:
+      'Full-stack developer. Dark mode enthusiast. Quality software devotee.',
+    // URL to your /uses page
+    url: 'https://www.linkedin.com/in/yuruiology/',
+    twitter: '@yuruiology',
+    // An emoji that describes you
+    emoji: '🐙',
+    // emoji of your country's flag
+    country: '🇨🇳',
+    // ONE of: apple, windows or linux
```
Comment (HugoDF) [inline src/data.js]:
```suggestion
```

**Comment by pallymore** [comment]
Fixed - thanks @HugoDF

**Pull request merged**